### ISSUE TITLE
Add hook to hold profile

### ISSUE THREAD

**Issue by friedger** [fungible-systems/micro-stacks#152]
UserData contains the profile url, however, it would be nice to have a hook to that contains the actual profile fetched from the profile url.

**Comment by aulneau** [comment]
Yeah, was thinking about what to do with the profile -- can you say more about use case?

**Comment by friedger** [comment]
@aulneau main use case for this is to use it on a Profile component of my app, e.g. my last transactions, my last name orders, etc.
